src/permapages/img/stamps/api.js

```diff
@@ -9,19 +10,23 @@ export async function getStamps(address) {
       const inputTag = JSON.parse(
         stamp["tags"].find((tag) => tag.name === "Input")?.value
       );
-      feed.push({
-        stampedAsset: inputTag?.transactionId,
-        stamper: stamp["tags"].find((tag) => tag.name === "Sequencer-Owner")
-          ?.value,
-        timestamp: inputTag?.timestamp,
-      });
+
+      if (inputTag.function === 'stamp') {
+        feed.push({
+          stampedAsset: inputTag?.transactionId,
+          stamper: stamp["tags"].find((tag) => tag.name === "Sequencer-Owner")
+            ?.value,
+          timestamp: inputTag?.timestamp,
+        });
+      }
     }
 
     if (address) {
       return feed.filter((stamp) => stamp.stamper === address);

```

Comment:
Can you please, inside this code block replace the `feed.filter()` with your `uniqBy` addition? Also to keep a return `feed` function incase no address was supplied (`address` is optional)

Reply:
Yep